project.clj

```diff
@@ -1,7 +1,7 @@
 (def ks-version "1.3.0")
 (def logback-version "1.1.3")
 
-(defproject puppetlabs/trapperkeeper "1.3.3-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper "1.4.0"
```

Comment:
If you're going to ship it with a jenkins job, i think you want this to be 1.4.0-SNAPSHOT